src/components/profile/profile-change-info/profile-change-info.ts

```diff
@@ -19,91 +19,112 @@ class ProfileChangeInfo extends Block {
         type: 'email',
         name: 'email',
         placeholder: 'pochta@yandex.ru',
         required: true,
-        pattern: regexpEmail,
+        pattern: REGEXP_EMAIL,
       },
-      classes: 'profile__field',
       validate: true,
     });
 
-    this.childrens.login = new FormField({
+    this.childrens.loginField = new FormField({
+      classes: 'profile__field',
       inputProps: {
         type: 'text',
         name: 'login',
         placeholder: 'ivanivanov',
         required: true,
         minlength: 3,
         maxlength: 20,
-        pattern: regexpLogin,
+        pattern: REGEXP_LOGIN,
       },
-      classes: 'profile__field',
       validate: true,
     });
 
-    this.childrens.firstName = new FormField({
+    this.childrens.firstNameField = new FormField({
+      classes: 'profile__field',
       inputProps: {
         type: 'text',
         name: 'first_name',
         placeholder: 'Иван',
         required: true,
-        pattern: regexpName,
+        pattern: REGEXP_NAME,
       },
-      classes: 'profile__field',
       validate: true,
     });
 
-    this.childrens.secondName = new FormField({
+    this.childrens.secondNameField = new FormField({
+      classes: 'profile__field',
       inputProps: {
         type: 'text',
         name: 'second_name',
         placeholder: 'Иванов',
         required: true,
-        pattern: regexpName,
+        pattern: REGEXP_NAME,
       },
-      classes: 'profile__field',
       validate: true,
     });
 
-    this.childrens.displayName = new FormField({
+    this.childrens.displayNameField = new FormField({
+      classes: 'profile__field',
       inputProps: {
         type: 'text',
         name: 'display_name',
         placeholder: 'Иван',
         required: true,
-        pattern: regexpName,
+        pattern: REGEXP_NAME,
       },
-      classes: 'profile__field',
       validate: true,
     });
 
-    this.childrens.phone = new FormField({
+    this.childrens.phoneField = new FormField({
+      classes: 'profile__field',
       inputProps: {
         type: 'tel',
         name: 'phone',
         placeholder: '+7 (909) 967 30 30',
         required: true,
-        pattern: regexpPhone,
+        pattern: REGEXP_PHONE,
       },
-      classes: 'profile__field',
       validate: true,
     });
 
+    this.childrens.avatarInput = new Input({
+      type: 'file',
+      name: 'avatar',
+      accept: 'image/*',
+      events: {
+        change: (event) => this.changeHandler(event),
+      },
+    });
+
     this.childrens.saveButton = new Button({
+      classes: 'profile__button',
       type: 'submit',
       name: 'Сохранить',
       text: 'Сохранить',
-      classes: 'profile__button',
     });
   }
 
+  changeHandler(event: Event) {
+    event.preventDefault();
+
+    const input = event.target as HTMLInputElement;
+    const formData = new FormData();
+    formData.append('avatar', input.files![0]);
+
+    UserController.updateAvatar(formData);
+  }
```

Comment:
Usercontroller.updateAvatar - асинхронная операция

```suggestion
  async changeHandler(event: Event) {
    event.preventDefault();

    const input = event.target as HTMLInputElement;
    const formData = new FormData();
    formData.append('avatar', input.files![0]);

    await UserController.updateAvatar(formData);
  }
```